### PR TITLE
feat: additional template event tracking

### PR DESCRIPTION
**Pull request by nickoferrall**
Fix https://github.com/ParabolInc/parabol/issues/7444

Demo: https://www.loom.com/share/80c0f4e1f34249dba68bcfafe101a6ae

### To test

- [ ] Open the poker or retro template picker and see the `Opened Template Picker` event is tracked and we're sending the  meeting type & scope as variables
- [ ] Click on a poker or retro template item and see the `Viewed Template` event is tracked and we're sending the meeting type, scope, and template name variables

### CODE REVIEW

File: packages/client/modules/meeting/components/PokerTemplateItem.tsx [hunk 1]
```diff
@@ -94,6 +95,14 @@ const PokerTemplateItem = (props: Props) => {
       store.get(teamId)?.setValue(null, 'editingScaleId')
     })
   }
+  useEffect(() => {
+    if (!isActive) return
+    SendClientSegmentEventMutation(atmosphere, 'Viewed Template', {
+      meetingType: 'poker',
+      scope,
+      templateName
+    })
```
Comment (tghanken) [inline packages/client/modules/meeting/components/PokerTemplateItem.tsx:104]: +1 I'd love to add `isFree` to the parameters we're sending to Segment.

File: packages/client/modules/meeting/components/ReflectTemplateItem.tsx [hunk 2]
```diff
@@ -64,14 +65,22 @@ interface Props {
...
   const selectTemplate = () => {
     setActiveTemplate(atmosphere, teamId, templateId, 'retrospective')
   }
+  useEffect(() => {
+    if (!isActive) return
+    SendClientSegmentEventMutation(atmosphere, 'Viewed Template', {
+      meetingType: 'retrospective',
+      scope,
+      templateName
+    })
```
Comment (tghanken) [inline packages/client/modules/meeting/components/ReflectTemplateItem.tsx:82]: +1 Would love to see `isFree` here as well

**Review by tghanken — APPROVED** [review]
LGTM.  Would love to add `isFree` to the Segment parameters, but that's not blocking.

File: packages/client/modules/meeting/components/PokerTemplateItem.tsx (hunk 1 above)
Comment (nickoferrall) [reply packages/client/modules/meeting/components/PokerTemplateItem.tsx]: Sure thing - added!

**Review by Dschoordsch — APPROVED** [review]
Did smoke test

**Pull request merged**